testing/src/test/java/io/stargate/it/http/graphql/schemafirst/SelectTest.java

```diff
@@ -47,73 +47,189 @@ public static void setup(StargateConnectionInfo cluster, @TestKeyspace CqlIdenti
             + "  cc1: Int! @cql_column(clusteringOrder: ASC)\n"
             + "  cc2: Int! @cql_column(clusteringOrder: DESC)\n"
             + "}\n"
+            + "type SelectFooResult @cql_payload {\n"
+            + "  data: [Foo]\n"
+            + "  pagingState: String\n"
+            + "}\n"
             + "type Query {\n"
             + "  foo(pk1: Int!, pk2: Int!, cc1: Int!, cc2: Int!): Foo\n"
-            + "  fooPartial(pk1: Int!, pk2: Int!, cc1: Int!): [Foo]\n"
+            + "  fooByPkAndCc1(pk1: Int!, pk2: Int!, cc1: Int!): [Foo]\n"
+            + "  fooByPk(pk1: Int!, pk2: Int!): [Foo]\n"
+            + "  fooByPkLimit(\n"
+            + "    pk1: Int!,\n"
+            + "    pk2: Int!\n"
+            + "  ): [Foo] @cql_select(limit: 5)\n"
+            + "  fooByPkPaginated(\n"
+            + "    pk1: Int!,\n"
+            + "    pk2: Int!,\n"
+            + "    pagingState: String @cql_pagingState\n"
```

Comment:
Shouldn't we consider implementing paging according to the `Relay` specification? 
i.e as discussed in the `production-ready grapqhl book`:
```
type PageInfo {
  endCursor: String
  hasNextPage: Boolean!
  hasPreviousPage: Boolean!
  startCursor: String
}

```
It may be simpler to integrate for clients that are already familiar with the `Relay`. I am not sure how widely it is adopted. wdyt?

Reply:
Yes, this is only a first version, but we could have other pagination options:

Relay pagination is on the roadmap for CQL-first (#142), and I agree that we should have it in GraphQL-first too.

We could also add a more traditional offset-based variant, like [OffsetPager](https://docs.datastax.com/en/drivers/java/4.10/com/datastax/oss/driver/api/core/paging/OffsetPager.html) in the Java driver.